server/sonar-process/src/main/java/org/sonar/process/ProcessProperties.java

```diff
@@ -48,15 +49,16 @@
   public static final String PATH_TEMP = "sonar.path.temp";
   public static final String PATH_WEB = "sonar.path.web";
 
+  public static final String SEARCH_CLUSTER_NAME = "sonar.search.clusterName";
```

Comment:
@simonbrandhof why isn't this property called `sonar.cluster.search.clusterName` ?

this property only makes sense if SQ runs as a cluster, therefor it should be prefixed with `sonar.cluster`, right?